configure.ac

```diff
@@ -2455,6 +2455,8 @@ AC_CHECK_FUNCS([endutxent getutxent getutxid getutxline getutxuser pututxline])
 AC_CHECK_FUNCS([setutxdb setutxent utmpxname])
 dnl    Checks for lastlog functions
 AC_CHECK_FUNCS([getlastlogxbyname])
+dnl    Various system functions
+AC_CHECK_FUNCS([mmap])
```

Comment:
Test that aren't platform specific should go in the big alphabetized AC_CHECK_FUNCS list where possible.